src/content/docs/enterprise/translation-process/crowdin-ai.mdx

```diff
@@ -40,7 +40,7 @@ To configure AI prompts, follow these steps:
   2. In the **Prompts** tab, click **New prompt**. Alternatively, you can go ahead with creating prompt samples clicking **Try a sample**. Crowdin allows you to quickly set up pre-configured prompts that are ready to perform effectively from the moment they are created. While these samples are designed to work good initially, you can customize them to your personal preferences.
   3. In the appeared dialog, set the prompt parameters:
       * Select **Enabled**
-      * Select the prompt type (i.e., Pre-translate, AI in editor, Alignment (available only for the Business [subscription plan](https://crowdin.com/pricing#enterprise-plans)), or Create custom type)
+      * Select the prompt type (i.e., Pre-translate, AI in editor, [Alignment](#ai-alignment) (available only for the Business [subscription plan](https://crowdin.com/pricing#enterprise-plans)), or Create custom type)
```

Comment:
We already mention the subscription plan below in the AI Alignment section, so there is no need to mention it here also.

Reply:
Since we're adding a link for the `Alignment` type, it's fair to add links for the `Pre-translate` and `AI in editor` types.